substrate/frame/election-provider-multi-block/src/verifier/tests.rs

```diff
@@ -480,26 +502,78 @@ mod async_verification {
 			MockSignedNextSolution::set(None);
 			MockSignedNextScore::set(None);
```

Comment:
In this test, we want to preserve the score, and actually see that a missing page still gets verified. 

atm this test is a duplicate of `solution_data_provider_missing_score_at_end`.